### ISSUE TITLE
[a11y-defect-3]: Benefit Letters breadcrumb links must describe their purpose and behave accordion to VADS specification

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#96391]
### Point of contact

Trevor Pierce

### Severity level

0, Launchblocker. Potentially Harmful. Must be fixed immediately.

### Details

The Benefits Letters breadcrumbs (static page logged in and logged out, and React app) have some inconsistencies. Specifically:

1. The static page is a bit misleading in its title. Users aren't downloading letters here, just logging in and getting relevant information.
2. The logged in Benefit Letters form view has the terminal (last) breadcrumb anchor linked to content

### Reproduction steps

1. Navigate to the static VA benefit letters page ( https://staging.va.gov/records/download-va-letters/ )
2. Read through the page and verify there's no actual downloads happening on this view before or after logging in.
3. Log in with a test user (I used 54) and navigate to the Benefit Letters form flow
4. Click on the last breadcrumb and verify it "jumps" down to the `#content` anchor

### Proposed solution or next steps

The CAIA team is working on a recommended set of breadcrumb behaviors that should be published soon. Please refer to those recommendations for how the React app crumb should behave. I'd recommend working with Content to explore renaming the static [Download VA benefit letters](https://staging.va.gov/records/download-va-letters/) page.

### References, articles, or WCAG support

1. Static [Download VA benefit letters page](https://staging.va.gov/records/download-va-letters/)
2. [Update IA Guidance: Breadcrumbs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73451)
3. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)
4. Understanding [SC 2.4.4](https://w3.org/TR/WCAG22#link-purpose-in-context): Link Purpose (In Context) (Level A)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [x] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [x] Markup and metadata
- [ ] Non-text content and media
- [x] Overall user experience
- [x] Cognitive | user flows & navigating
- [x] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
We will wait for the guidance on breadcrumbs to be finalized and published before we action on this to avoid rework.

**Comment by dysbo** [comment]
Breadcrumb looks as expected!

<img width="834" height="484" alt="Image" src="https://github.com/user-attachments/assets/3642d7b2-0fd4-460c-9d23-7c4246ad5b27" />
